src/app/globals.css

```
@@ -4608,14 +4608,917 @@ html.custom-cursor-active * {
+   above it — measured to fit at 760) and 375px (the shortest landscape phone
+   in common use), where the stage lands at half size. Everything between is
+   the straight line through those two points; `svh` rather than `vh` so an
+   iOS toolbar cannot make the page taller than the screen. */
+
+/* The headline ladder. Values identical to the utilities they replaced; the
+   `min()` is what the vertical-fit block below reaches. `--title-h` /
+   `--role-h` are unset by default, and the 9999px fallback is what makes the
+   cap inert until a short viewport declares one. */
+h1.home-title {
+  --title-w: 2.6rem;
+  font-size: min(var(--title-w), var(--title-h, 9999px));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)